VisualPinball.Engine.Test/VisualPinball.Engine.Test.csproj

```diff
@@ -15,16 +15,13 @@
   </PropertyGroup>
   <ItemGroup>
     <None Remove="**/*.meta" Visible="false" />
-    <None Remove=".bin/*" Visible="false" />
-    <None Remove=".obj/*" Visible="false" />
-    <None Remove="obj/*" Visible="false" />

```

Comment:
Not needed anymore?

Reply:
Correct. In Rider or VS, I'm not seeing those directories. 

![Screen Shot 2020-07-08 at 7 04 33 AM](https://user-images.githubusercontent.com/1197137/86911474-64d7d200-c0e9-11ea-8968-2b772e752556.png)

The only way I see them in Rider is if I have it listed to `File System` instead of `Solution`:

![Screen Shot 2020-07-08 at 7 04 53 AM](https://user-images.githubusercontent.com/1197137/86911551-82a53700-c0e9-11ea-830a-313b4ec0b6f0.png)

I think the missing backslash at the end of `.obj` in `Directory.build.props` may have fixed this?